base_import_pdf_by_template_account/demo/base_import_pdf_template.xml

```diff
@@ -152,4 +159,18 @@
         <field name="value_type">variable</field>
         <field name="log_distinct_value" eval="True" />
     </record>
+    <record
+        id="invoice_tecnativa_analytic_distribution"
+        model="base.import.pdf.template.line"
+    >
+        <field name="template_id" ref="invoice_tecnativa" />
+        <field name="related_model">lines</field>
+        <field
+            name="field_id"
+            ref="account.field_account_move_line__analytic_distribution"
+        />
+        <field name="column">3</field>
+        <field name="value_type">fixed</field>
+        <!--<field name="fixed_value_text">{"cls.env.ref('base_import_pdf_by_template_account.analytic_account_tecnativa').id": 100.0}</field>!-->
```

Comment:
It is not necessary?

Reply:
Yes, it is necessary but it is commented because it is not possible to do that directly. If you know the way to do it, it will be welcome! (this is currently set correctly in tests).